Refactor Enterprise Operations Manual for consistency and readability (#2276)
travis-ci/docs-travis-ci-com: user/enterprise/platform-tips.md

If everything is up and running, it answers with a `HTTP 200 OK`, or in case of failure with a `HTTP 500 Internal Server Error`.

> ## Configuring Backups

Author: I mentioned this in the commit message but I feel that this fits better in this file than the Operations Manual/Troubleshooting Guide. This section isn't an 'issue', really, and the rest of the troubleshooting guide contains instructions on how to fix issues.

Reviewer: :+1:

Reviewer 2: 👍

This section explains how you integrate Travis CI Enterprise in your backup strategy. Here, we'll talk about two topics:

> - The encryption key

Reviewer: probably link these, although it doesn't make much odds here.

Author: Addressed in https://github.com/travis-ci/docs-travis-ci-com/pull/2276/commits/02b82bf917aefc2d9c30980be9662f42c5b3a8e4

- The data directories

### Encryption key

Without the encryption key you cannot access the information in your production database. To make sure that you can always recover your database, make a backup of this key.

> Without the encryption key the information in the database is not recoverable.

To make a backup, please follow these steps:

> 1. open a ssh connection to the platform machine

Reviewer: mostly we use full sentences for lists I think. But especially be consistent with capitalization and punctuation throughout the list.

Author: Addressed in https://github.com/travis-ci/docs-travis-ci-com/pull/2276/commits/02b82bf917aefc2d9c30980be9662f42c5b3a8e4

> 2. run `travis bash`. This will open a bash session with `root` privileges into the Travis container.

Reviewer: These often make more sense when you say what/why you're doing first, then how.

Author: Addressed in both your commit and my latest: https://github.com/travis-ci/docs-travis-ci-com/pull/2276/commits/02b82bf917aefc2d9c30980be9662f42c5b3a8e4

> 2. run `travis bash`. This will open a bash session with `root` privileges into the Travis container.

Reviewer: ```suggestion
2. Open a bash session  with `root` privileges on the Travis CI container by running `travis bash`.
```

> 3. Then run `grep -A1 encryption: /usr/local/travis/etc/travis/config/travis.yml`. Create a backup of the value returned by that command by either writing it down on a piece of paper or storing it on a different computer.

Reviewer: And  here you could split into 2 steps

1. find the encrypt key
2. Make a backup

Author: Addressed in https://github.com/travis-ci/docs-travis-ci-com/pull/2276/commits/02b82bf917aefc2d9c30980be9662f42c5b3a8e4

### Create a backup of the data directories

The data directories are located on the platform machine and get mounted into the Travis container. In these directories you'll find files from RabbitMQ, Postgres, Slanger, Redis and also log files from the applications inside the container.

The files are located at `/var/travis` on the platform machine. Please run `sudo tar -czvf travis-enterprise-data-backup.tar.gz /var/travis` to create compressed archive from this folder. After this has finished, copy this file off the machine to a secure location.

> 

Reviewer: Too much to mention backup rotations etc here probably?

Author: I am not sure, to be honest, but that's mainly because I haven't yet learned what our exact backup rotation rules are. 😬 

I would prefer to leave this untouched for now in the interests of minimizing scope if you are okay with it. What do you think?

Reviewer: Sure!
